pkg/controller/networkpolicy/crd_utils.go

```diff
@@ -132,9 +134,22 @@ func (n *NetworkPolicyController) createAddressGroupForCRD(peer secv1alpha1.Netw
 // getTierPriority retrieves the priority associated with the input Tier name.
 // If the Tier name is empty, by default, the lowest priority Application Tier
 // is returned.
-func getTierPriority(tier string) controlplane.TierPriority {
+func (n *NetworkPolicyController) getTierPriority(tier string) int32 {
 	if tier == "" {
-		return antreatypes.TierApplication
+		return defaultTierPriority
+	}
+	// If the tier name is part of the static tier name set, we need to convert
+	// tier name to lowercase to match the corresponding Tier CRD name. This is
+	// possible in case of upgrade where in a previously created Antrea Policy
+	// CRD was referring to an old static tier.
```

Comment:
Would you add the old Antrea version number that requires the conversion here?

Reply:
done